### DISCUSSION TITLE
Token Representation

### DISCUSSION THREAD

**Discussion by TheusBoot** [fastapi/fastapi#7273] [Questions]
### Token Representation

- [x] FastAPI could have a greater focus on tokens, so it would open the way for this tool to be used in companies, I am currently learning it, but I cannot apply it to my team due to the lack of a well formulated description about authentication

**Comment by Kludex** [comment]
Search for "authentication" on our documentation...

**Comment by tiangolo** [comment]
Thanks for the help @Kludex ! :rocket: 

@TheusBoot if you still have problems, please create a new issue and follow the template step by step. That way it's easier to help you with your specific problem. Given that, I'm gonna close this issue now.
